jobs/build/custom/Jenkinsfile

```diff
@@ -143,6 +149,11 @@ node {
     def exclude_images = commonlib.cleanCommaList(params.EXCLUDE_IMAGES)
     def rpms = commonlib.cleanCommaList(params.RPMS)
 
+    def runtime_assembly = null
+    def assemblies_enabled = buildlib.doozer("${doozerOpts} config:read-group --default=False assemblies.enabled", [capture: true]).trim() == 'True'
```

Comment:
It is just to determine we should put plashets to`puddles/RHAOS/plashets/{MAJOR}.{MINOR}-el8/` or `puddles/RHAOS/plashets/{MAJOR}.{MINOR}-el8/{runtime_assembly}/`. The path is used in `buildlib.buildBuildingPlashet()`.

If we remove this variable, we need to have a symlink from the legacy path to `stream/building_embargoed` for all ocp versions, even if assembly is not enabled.

Reply:
@sosiouxme Well, I just realized having a symlink doesn't solve everything. If assemblies are disabled and a user starts a build with `ASSEMBLY=test`, rpms will be put into `puddles/RHAOS/plashets/{MAJOR}.{MINOR}-el8/test/` but Doozer will not be able to consume it. The problem is currently plashet paths are populated by Jenkinsfiles, not Doozer.

Reply:
Perhaps we can just change the check to prevent that human error? If assemblies are not enabled and you find ASSEMBLY!=stream, error('') the job. The legacy symlink should then always work as expected.